src/data_structures/bitenc.rs

```diff
@@ -19,7 +19,7 @@
 //! ```
 
 /// A sequence of bitencoded values.
-#[derive(Serialize, Deserialize)]
+#[derive(Serialize, Deserialize, PartialEq, Hash)]
```

Comment:
```suggestion
#[derive(Serialize, Deserialize, PartialEq, Eq, Hash)]
```

Reply:
I tried this a first. But when adding BitEnc as field type into another struct the compiler was not satisfied, complaining about std::cmp::Eq being missing.